_specs/ecip-1045.md

```diff
@@ -2,7 +2,7 @@
 lang: en
 ecip: 1045
 title: Support for ETH Byzantium & Constantinople EVM and Protocol Upgrades
-status: Draft
+status: Withdrawn
```

Comment:
```suggestion
status: Rejected
```

Reply:
Replaced again here?
By https://ecips.ethereumclassic.org/ECIPs/ecip-1054 and https://ecips.ethereumclassic.org/ECIPs/ecip-1056.